website/docs/sidebar.md

```diff
@@ -77,9 +77,7 @@ module.exports = {
 };
 ```
 
-:::note
-Shorthand notation relies on the iteration order of JavaScript object keys for the category name. When using this notation, keep in mind that EcmaScript does not guarantee `Object.keys({a,b}) === ['a','b']`, yet this is generally true.
-:::
+:::note Shorthand notation relies on the iteration order of JavaScript object keys for the category name. When using this notation, keep in mind that EcmaScript does not guarantee `Object.keys({a,b}) === ['a','b']`, yet this is generally true. :::
```

Comment:
@josephriosIO - same here, revert this. There needs to be newlines after and before the `:::`